bot/exts/info/information.py

```diff
@@ -228,6 +228,9 @@ async def create_user_embed(self, ctx: Context, user: FetchedMember) -> Embed:
         if on_server and user.nick:
             name = f"{user.nick} ({name})"
 
+        if user.bot:
+            name += f" {constants.Emojis.bot}"
```

Comment:
If we truly want to have the bot emoji next to the name, it should be in same style as it is natively in discord, if bot is verified, give it verified emoji, if not, bot emoji. Putting bot emoji next to the name and verified badge below it is just weird.

Reply:
It has been moved back to the title